packages/@react-types/avatar/src/index.d.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2021 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {DimensionValue, DOMProps, StyleProps} from '@react-types/shared';
+
+export interface AvatarProps {
+  /**
+   * Text description of the avatar.
+   *
+   * @default null
+   */
+   alt?: string,
+   /**
+    * The image URL for the avatar.
+    */
+   src: string
```

Comment:
technically this is optional https://spectrum.adobe.com/page/avatar/#-Use-generic-avatars-when-image-is-undefined

Reply:
Ah yep, we interpreted it two different ways. Let's get some more opinions.
Down sides of both:
- optional image src means we would need to embed some images by default or code it to a discoverable place that we control so it can't be changed without us knowing
- required img src means potential for people forgetting to implement the generic icon or getting it from the wrong place